cogs/source/postSource.py

```diff
@@ -0,0 +1,53 @@
+import tracemoepy
+from tracemoepy.errors import EmptyImage, EntityTooLarge, ServerError, TooManyRequests
+
+
+async def postSourceFunction(self, ctx, imageURL):
```

Comment:
As per our discussion, can we please move this back into the other file, keep it outside of the class (since no reference to self), and use it?

```py
async def postSource(ctx, imageUrl):
    # Do stuff.


class Source(commands.Cog):
    # ...
    async def source(self, ctx):
        # Some discord related stuff.
        await postSource(ctx, imageUrl)
    # ...
    async def sourceUrl(self, ctx, imageUrl):
        # Some discord related stuff.
        await postSource(ctx, imageUrl)
```

Reply:
This should be done now